docusaurus/docs/dev-docs/deployment/strapi-cloud.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Strapi Cloud
+displayed_sidebar: devDocsSidebar
+description: Learn how to deploy your Strapi application on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.html
```

Comment:
Will the canonicalUrl change as well?

Reply:
Yes, we will have to change the canonicalUrl properties as well. I have a dedicated task on my to-do list for this. I think it would be better to change them only when we have the new URL (docs.strapi.io) in place, but I'll double-check.